### PR TITLE
KN protocol 12 channels

### PR DESCRIPTION
**Pull request by fgiudice98**
The V977 stock transmitter send all (working) trims (T, A, E, R). This is to make the Deviation perfectly the same as a stock V977 transmitter

### CODE REVIEW

File: src/protocol/kn_nrf24l01.c [hunk 1]
```diff
@@ -562,7 +563,7 @@ static void kn_update_packet_control_data(u8 packet[], s32 packet_count, s32 rf_
     } else {
        packet[9]  = Channels[CHANNEL10] /101 + 100; // 0x64; // A
        packet[10] = Channels[CHANNEL11] /101 + 100; // 0x64; // E
-       packet[11] = 0x64; // R
+       packet[11] = Channels[CHANNEL12] /101 + 100; // 0x64; // R
```
Comment (PhracturedBlue) [inline src/protocol/kn_nrf24l01.c]: Lint Error:
```
566:  At least two spaces is best between code and comments  
```

Comment (fgiudice98) [reply src/protocol/kn_nrf24l01.c]: My line is the same as the existing ones

Comment (goebish) [reply src/protocol/kn_nrf24l01.c]: That's because there was no Lint check yet when this file was first commited.

Comment (fgiudice98) [reply src/protocol/kn_nrf24l01.c]: What to do then? Should change all the file commets?

Comment (goebish) [reply src/protocol/kn_nrf24l01.c]: At least the one on the line you modified

**Comment by fgiudice98** [comment]
Should be good now

File: src/protocol/kn_nrf24l01.c [hunk 2]
```diff
@@ -235,7 +236,7 @@ static void kn_start_tx(BOOL bind_yes)
     kn_init(tx_addr_, hopping_channels_);
     if(bind_yes)
     {
-        PROTOCOL_SetBindState(bind_count * BINDING_PACKET_PERIOD / 1000); //msec
+        PROTOCOL_SetBindState(bind_count * BINDING_PACKET_PERIOD / 1000);  //msec
```
Comment (PhracturedBlue) [inline src/protocol/kn_nrf24l01.c]: Lint Error:
```
239:  Should have a space between // and comment  
```

File: src/protocol/kn_nrf24l01.c [hunk 3]
```diff
@@ -295,7 +296,7 @@ static s32 rf_ch_idx = 0;
         }
         packet_sent++;
         return sending_packet_period;
-    } //switch
+    }  //switch
```
Comment (PhracturedBlue) [inline src/protocol/kn_nrf24l01.c]: Lint Error:
```
299:  Should have a space between // and comment  
```

**Pull request merged**